ibmcloud/README.md

```diff
@@ -273,31 +362,33 @@ $ install cloud-api-adaptor /usr/local/bin/
 
 ## Launch Cloud API adaptor
 
-A terraform template that will start the `cloud-api-adaptor` process on the Kubernetes worker node is available in [ibmcloud/terraform/start-cloud-api-adaptor](./terraform/start-cloud-api-adaptor).
+A terraform configuration that will start the `cloud-api-adaptor` process on the Kubernetes worker node is available in [ibmcloud/terraform/start-cloud-api-adaptor](./terraform/start-cloud-api-adaptor).
 
-Create a `terraform.tfvars` file in the [template directory](./terraform/start-cloud-api-adaptor) for this Terraform template on your `development machine`. The `terraform.tfvars` file should look like this
+Create a `terraform.tfvars` file in the [configuration directory](./terraform/start-cloud-api-adaptor) for this Terraform configuration on your `development machine`. The `terraform.tfvars` file should look like this
 
 ```
 ibmcloud_api_key = "<your API Key>"
 cluster_name = "<cluster name>"
-ssh_key_name = "<your SSH key name>"
-podvm_image_name = "<name of your pod VM image>"
+ssh_key_name = "<your SSH key name>" OR ssh_key_id = "<your SSH key id>"
+podvm_image_name = "<name of your pod VM image>" OR podvm_image_id = "<ID of your peer pod VM image>"
+vpc_name = "<name of your VPC>" OR vpc_id = "<ID of your VPC>"
+primary_subnet_name = "<name of your primary subnet>" OR primary_subnet_id = "<ID of your primary subnet>"
+primary_security_group_name = "<name of your primary security group>" OR primary_security_group_id = "<ID of your primary security group>"
 ```
```

Comment:
Optional/idea: I wonder if it is worth highlighting to the user that the `podvm_image_id` is provided as an output variable in the **Build a pod VM image** section and the `vpc_id`, `primary_subnet_id` and `primary_security_group_id` are output variables in the **Create a VPC section**?

Reply:
I did the same thing here as I did in the https://github.com/confidential-containers/cloud-api-adaptor/pull/49#discussion_r893355021 comment